.github/workflows/update_prebuilts.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow creates a new PR with the updated build ids for Playground prebuilts.
+# To run it, navigate to https://github.com/androidx/androidx/actions/workflows/update_prebuilts.yml
+# and click "Run workflow"
+name: Update Prebuilts
+on:
+  workflow_dispatch:
+    inputs:
+      target-branch:
+        description: 'The target branch for the PR'
+        required: false
+        default: 'androidx-main'
+        type: string
+      comment:
+        description: 'Any comment you would like to add to the PR body'
+        required: false
+        default: ''
+        type: string
+jobs:
+  update-prebuilts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout androidx repo"
+        uses: actions/checkout@v2
+      - name: "Set environment variables"
+        shell: bash
```

Comment:
I know we do this elsewhere, but now we can use https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrun so we don't have to keep copying this around.